charts/model-engine/templates/service_template_config_map.yaml

```diff
@@ -572,7 +572,7 @@ data:
                 {{- tuple $env_var | toYaml | nindent 16 }}
                 {{- end }}
                 {{- end }}
-              imagePullPolicy: Always
+              imagePullPolicy: IfNotPresent
```

Comment:
@seanshi-scale let me know this change is okay

Reply:
This should be fine, think our image tags are immutable